joy_song/about_delete.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang = "en-US">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css" />
+        <link rel="stylesheet" href="style1.css">
+        <title> About Me</title>
+    </head>
+
+    <body>
+        <nav class="navbar" style="background-color: #c1cefe" id="content">
+            <div class="container">
+                <div class="navbar-brand">
+                    <a class="navbar-item brand-text" href="about.html">
```

Comment:
Make sure to align your code. In industry, you will have something called a linter that throws errors when there are formatting violations in your code.